tensorflow/models/image/mnist/convolutional.py

```diff
@@ -296,11 +296,13 @@ def eval_in_batches(data, sess):
       # node in the graph it should be fed to.
       feed_dict = {train_data_node: batch_data,
                    train_labels_node: batch_labels}
-      # Run the graph and fetch some of the nodes.
-      _, l, lr, predictions = sess.run(
-          [optimizer, loss, learning_rate, train_prediction],
-          feed_dict=feed_dict)
+      # Run the optimizer to update weights.
+      sess.run(optimizer, feed_dict = feed_dict)
```

Comment:
Does omitting the three fetches really make that much of a difference? They should have been computed already so the only overhead should be the transfer from the session to numpy arrays.